log-parser-plugin-rules.txt

```diff
@@ -0,0 +1,66 @@
+start /\sPLAYBOOK:\s/
+start /^\./run.sh/
+
+info /\+\sansible-playbook\s/
+info /\sTASK\s\[/
+info /skipped/
+
+# set the retries as warning so we can check them faster
+warning /\sFAILED - RETRYING:/
```

Comment:
This will be a lot of warnings, but it will bring stats/trends on how we deal with retries, so I like this.